arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/TimestampUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.arrow.vectorized.parquet;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.TimeUnit;
+
+public class TimestampUtil {
+
+  private TimestampUtil() {}
+
+  private static final long UNIX_EPOCH_JULIAN = 2_440_588L;
+
+  public static long extractTimestampInt96(ByteBuffer buffer) {
```

Comment:
can we adjust Sparks version of this in `TimestampInt96Reader` to reuse `TimestampUtil`?

Reply:
I replaced all timestampint96 related places, please take a look~

Reply:
this feels like something that could live in `ParquetUtil`, instead of creating a new class?